engine/src/main/java/com/redhat/cloud/notifications/processors/camel/CamelTypeProcessor.java

```diff
@@ -257,6 +271,9 @@ private void callOpenBridge(JsonObject body, UUID id, String orgId, CamelPropert
                 .baseUri(URI.create(bridge.getEndpoint()))
                 .build(BridgeEventService.class);
 
+        // Include the environment's URL when sending the event.
+        ce.put("environment_url", this.environment.url());
```

Comment:
Even though this _could_ work (I'm not sure about it), let's put the `environment_url` in the `body` variable of this method (after `body.remove(NOTIF_METADATA_KEY)` for example) so that the URL ends up in the `data` field that is used in RHOSE to generate the Slack notification from our template.

Reply:
All right! Done!